### PR TITLE
Update default.html

### PR DESCRIPTION
**Pull request by x-a-n-a-x**
Added meta name="keywords"

### CODE REVIEW

File: layouts/default.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=1024" />
     <meta name="description" content="Open Source reimplementation of Westwood Studios' 2D Command and Conquer games" />
+    <meta name="keywords" content="OpenRA, RTS, game engine, RTS game engine, strategy, mods, C&amp;C, Command and Conquer, C&amp;C95, C&quot;C1, C&amp;C95 Online, Online, LAN, multiplayer, Red Alert, Red, Alert, RA, RA1, RA95, Tiberian, Tiberian Dawn, TD, Dune, Dune 2000, Dune2000, D2K, Tiberium Sun, Tiberium, Sun, TS, Red Alert 2, RA2, Kane, GDI, Nod, The Brotherhood of Nod, The Global Defense Initiative, Atreides, Ordos, Harkonnen, Allies, Soviets, Westwood" />
```
Comment (Mailaender) [inline layouts/default.html]: It is `Tiberian Sun` not `Tiberium Sun`.

**Comment by x-a-n-a-x** [comment]
https://github.com/OpenRA/OpenRAWeb/issues/224

File: layouts/default.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=1024" />
     <meta name="description" content="Open Source reimplementation of Westwood Studios' 2D Command and Conquer games" />
+    <meta name="keywords" content="OpenRA, RTS, game engine, RTS game engine, strategy, mods, C&amp;C, Command and Conquer, C&amp;C95, C&quot;C1, C&amp;C95 Online, Online, LAN, multiplayer, Red Alert, Red, Alert, RA, RA1, RA95, Tiberian, Tiberian Dawn, TD, Dune, Dune 2000, Dune2000, D2K, Tiberian Sun, Tiberian, Tiberium, Sun, TS, Red Alert 2, RA2, Kane, GDI, Nod, The Brotherhood of Nod, The Global Defense Initiative, Atreides, Ordos, Harkonnen, Allies, Soviets, Westwood" />
```
Comment (Mailaender) [inline layouts/default.html]: `Tiberian` is now listed twice

**Comment by abcdefg30** [comment]
Hm, what about `Global Defense Initiative` and `Brotherhood of Nod` (without `The`).

And you also split e.g. `Red Alert` into `Red` and `Alert`. What about splitting above too then? ^^

File: layouts/default.html (hunk 2 above)
Comment (Mailaender) [inline layouts/default.html]: You could reduce the spam a bit:
> Red Alert, Red, Alert, [...]

The proper term is enough I guess.

**Pull request closed without merging**